src/main/resources/application.yml

```diff
@@ -3,3 +3,8 @@ spring:
     mongodb:
       uri: mongodb+srv://admin:Password1234@codeninjas.hkvxevj.mongodb.net/?retryWrites=true&w=majority
       database: eCommerce
+management:
+  endpoints:
+    web:
+      exposure:
+        include: "*"
```

Comment:
please also add configuration for health endpoint to show details

    endpoint:
      health:
        show-details: always

Reply:
will do boss